sirepo/job_supervisor.py

```diff
@@ -666,6 +679,7 @@ def _set_error(op, compute_job_serial):
             simName=req.content.data.models.simulation.name,
             status=job.PENDING,
         )
+        self._cache_purged_jid(self.db.computeJid, invalidate=True)
```

Comment:
```py
self._purged_jids_cache._discard(self.__db_file(self.db.computeJid).purebasename))
```

perhaps `self.__db_file()` should be an attribute of self so it is calculated once.

Reply:
https://github.com/radiasoft/sirepo/pull/2485/commits/3cae5f2b5b9cf09fc482886f2745bbeecd1421e6